components/script/dom/oscillatornode.rs

```diff
@@ -113,6 +115,19 @@ impl OscillatorNodeMethods for OscillatorNode {
     fn Detune(&self) -> DomRoot<AudioParam> {
         DomRoot::from_ref(&self.detune)
     }
+
+    // https://webaudio.github.io/web-audio-api/#dom-oscillatornode-type
+    fn Type(&self) -> OscillatorType {
+        self.oscillator_type.get()
+    }
+
+    // https://webaudio.github.io/web-audio-api/#dom-oscillatornode-type
+    fn SetType(&self, oscillator: OscillatorType) {
+        self.oscillator_type.set(oscillator);
+        self.node.message(AudioNodeMessage::OscillatorNode(
```

Comment:
`OscillatorNode` has no `node` member. In this case, we are inheriting from `AudioScheduledSourceNode` (which inherits from `AudioNode`). So what we have is a `source_node` member, which has a `node()` getter. This should be:

```rust
self.source_node.node().message(...)
```